src/module.c

```diff
@@ -8589,6 +8591,7 @@ void moduleNotifyKeyUnlink(robj *key, robj *val, int dbid) {
             mt->unlink(key,mv->value);
         } 
     }
+    moduleNotifyKeyspaceEvent(NOTIFY_UNLINK,"unlink",key,dbid);
```

Comment:
i think it's better to move this call to the top of the function, before any action is taken.
also, please update the comment. "when a module key is deleted", the word "module" needs to be removed.
```suggestion
    moduleNotifyKeyspaceEvent(NOTIFY_REMOVED,"removed",key,dbid);
```